provision/kubernetes/job_test.go

```diff
@@ -684,7 +685,9 @@ func (s *S) TestCreateJobEvent(c *check.C) {
 					Reason:  "SuccessfulCreate",
 					Type:    "Normal",
 				}
-				createJobEventAndMetrics(j, evt)
+				wg := &sync.WaitGroup{}
+				wg.Add(1)
+				createJobEvent(j, evt, wg)
```

Comment:
?? 
Actually no.. I didn't spawn a goroutine, there's no need to wg.Wait()... The only reason I added wg.Add(1) is because we decrement it at the defer statement